cli.js

```diff
@@ -142,6 +142,8 @@ process.on('uncaughtException', function(err) {
 
     flags = flags || [];
     optFlags = optFlags || [];
+    // we always take a log option
+    optFlags.push('log');
```

Comment:
Went with your code above for now; I think it probably requires a bit of a rework the next time we add a global option, but for now it seems overkill.